homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -453,7 +453,7 @@ def get_entity_state(config, entity):
     if cached_state is None:
         data[STATE_ON] = entity.state != STATE_OFF
         if data[STATE_ON]:
-            data[STATE_BRIGHTNESS] = entity.attributes.get(ATTR_BRIGHTNESS)
+            data[STATE_BRIGHTNESS] = entity.attributes.get(ATTR_BRIGHTNESS, 0)
```

Comment:
I think a better solution is to not set the brightness in the data at all if it's not in the attributes.

Reply:
Based on this comment https://github.com/home-assistant/home-assistant/issues/23514#issuecomment-493256539 in the issue, apparently not setting brightness really confuses Alexa.

I think for hue/saturation, that makes more sense, and I can make that change if you like

Reply:
This is the whole basis behind the issue.  Alexa expects a brightness and it's not getting it, so alexa assumes that the light isn't responding.  We need to set the brightness.

Reply:
You would expect that if you set the type as "On/Off light" that Alexa would not look for a brightness value at all (and not include a dimmer slider on Show devices) but that didn't seem true based on my brief testing yesterday.